src/styles/card.scss

```diff
@@ -0,0 +1,36 @@
+@import './stars';
+@import './colors';
+
+.card {
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+
+    display: block;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $bold-text-color;
+
+    margin-top: 40px;
+  }
```

Comment:
The `font-size`, `font-weight`, `line-height`, and `color` properties in `.card__title` should be alphabetically ordered.